src/sestest/ListCertificatesCommand.cs

```diff
@@ -96,6 +96,28 @@ private int Execute(ShowCertificates showCerts)
                                      && !c.SupportsUse(X509KeyUsageFlags.DataEncipherment)));
             }
 
+            if (showCerts == ShowCertificates.All
+                || showCerts == ShowCertificates.ForServerAuth
+                || showCerts == ShowCertificates.NonExpired)
+            {
+                var serverAuthCerts = candidates
+                    .Where(cert => now < cert.NotAfter && cert.SupportsServerAuthentication())
+                    .Select(cert => new
+                    {
+                        IsVerified = cert.Verify(),
+                        Cert = cert
+                    })
+                    .ToList();
+
+                LogCertificates(
+                    "Found {0} non-expired certificates with private keys that allow server authentication and are verified",
+                    serverAuthCerts.Where(c => c.IsVerified).Select(c => c.Cert));
+
+                LogCertificates(
+                    "Found {0} non-expired certificates with private keys that allow server authentication and are not verified",
```

Comment:
Suggest making this `"... and are NOT verified"` to make the visual distinction with the prior message more visible.